lib/client/browser.js

```diff
@@ -27,7 +37,9 @@ require("./discovery.js");
 				context._frame.onload = function() {
 					onload.call(null, context);
 				};
-				context._frame.setAttribute("style", "height: 1024px; width: 1280px;");
+				context._frame.style.height = "100%";
```

Comment:
Nice

As alternative it would be also possible to use Object.assign it's more personal preference 

```javascript
Object.assign(context._frame.style, {
    height: "100%",
    widht: "100%",
    border: "0"
})
```